src/App/index.tsx

```diff
@@ -22,6 +25,11 @@ const App = () => {
             {welcomeInstructions}
           </Welcome>
         )}
+        <OnScreenKeyboard
+          searchQuery={searchInput}
+          searchUpdateHandler={setSearchInput}
+          triggerSearchHandler={() => {console.log(`SEARCH: ${searchInput}`)}}
```

Comment:
Did you mean to leave in the `console.log` statement?

Reply:
Yeah, I think it's helpful to have some debugging feedback until we get the real search functionality in place. That'll be removed when Algolia's configured though.